docs/charts/area.md

```diff
@@ -29,16 +29,16 @@ new Chart(ctx, {
             {fill: '-2'}           // 4: fill to dataset 2
         ]
     }
-})
+});
 ```
 
 ## Configuration
 | Option | Type | Default | Description |
 | :--- | :--- | :--- | :--- |
-| [`plugins.filler.propagate`](#propagate) | `Boolean` | `true` | Fill propagation when target is hidden.
+| [`plugins.filler.propagate`](#propagate) | `boolean` | `true` | Fill propagation when target is hidden.
 
 ### propagate
-Boolean (default: `true`)
+A boolean value (default: `true`)
```

Comment:
Since the current wording is just a fragment, I might say something like:

`propagate` takes a `boolean` value (default: `true`)